config/initializers/forbidden_yaml.rb

```diff
@@ -1,46 +0,0 @@
-# XXX: This is purely a monkey patch to close the exploit vector for now, a more
-# permanent solution should be pushed upstream into rubygems.
-
-require "rubygems"
-
-# Assert we're using Psych
-abort "Use Psych for YAML, install libyaml and reinstall ruby" unless YAML == Psych
-
-module Gem
-  class Specification
-    WHITELISTED_CLASSES = %w(
-      Symbol
-      Time
-      Date
-      Gem::Dependency
-      Gem::Platform
-      Gem::Requirement
-      Gem::Specification
-      Gem::Version
-      Gem::Version::Requirement
-    )
-
-    WHITELISTED_SYMBOLS = %w(
-      development
-      runtime
-    )
-
-    def self.from_yaml(input)
-      input = normalize_yaml_input input
-      spec = Psych.safe_load(input, WHITELISTED_CLASSES, WHITELISTED_SYMBOLS, true)
```

Comment:
`safe_load` call is here. So this patch needs to stay.

default `Gem::Specification. from_yaml` will use just load. https://github.com/rubygems/rubygems/blob/master/lib/rubygems/specification.rb#L1061